mpt_extension_sdk/swo_rql/query_builder.py

```
@@ -34,22 +37,34 @@ def parse_kwargs(query_dict):
+    if isinstance(value, dt.date | dt.datetime):
+        return value.isoformat()
+    return None
+
+
+def _quote_comparison_value(op, value):
+    if op in constants.COMP:
+        return f"'{value}'"
+    return value
+
+
+def rql_encode(op, value):  # noqa: D103
```

Ruff (0.15.0)

[warning] Unused `noqa` directive (non-enabled: `D103`)

Remove unused `noqa` directive

(RUF100)